nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -1502,6 +1508,16 @@ GeckoVBufBackend_t::GeckoVBufBackend_t(int docHandle, int ID): VBufBackend_t(doc
 }
 
 GeckoVBufBackend_t::~GeckoVBufBackend_t() {
+	// The backend holds a reference to the root accessible of the document.
+	// This must be specifically released in the UI thread where it was created.
+	// See https://issues.chromium.org/issues/41487612
+	// In most cases this will be released in renderThread_terminate.
+	// However in the unlikely case terminate can't run,
+	// it will be deleted along with the backend, but in an RPC thread!
```

Comment:
This might be misconstrued to mean that even with the code below, it will be deleted from the wrong thread. Perhaps this could say "if we don't do anything here, it will be deleted along with..."